src/Components/Login/Login.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { withRouter, useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+
+function Login() {
+  const history = useHistory();
+  const KakaoLogin = () => {
```

Comment:
state, props등의 부분과 일반 함수를 선언하는 부분은 한 줄 공백 주는게 좋습니다!
```suggestion

  const KakaoLogin = () => {
```

Reply:
첨에 공백줬다가 뭔가 붙여야할거같아서 붙였는데 공백주는게 맞았군요 🤔🤔🤔